test/ModalSpec.js

```diff
@@ -11,6 +11,16 @@ import { render, shouldWarn } from './helpers';
 
 const $ = componentOrNode => jQuery(ReactDOM.findDOMNode(componentOrNode));
 
+
+class ErrorBoundary extends React.Component {
+  componentDidCatch(error, info) {
```

Comment:
Do you want to upgrade the React dev-dependencies to 16? Otherwise you want want to add support for `unstable_handleError`.